src/main/webapp/resources/js/pages/launch/LaunchFiles.jsx

```diff
@@ -60,7 +60,7 @@ export function LaunchFiles() {
         const firstSamples = [];
         data.forEach((sample) => {
           if (sample.files.length) {
-            sample.selected = sample.files[0].identifier;
+            sample.selected = sample.files[0].fileInfo.identifier;
```

Comment:
Is the plan to move this into redux tool-kit?  This might be it's own API.

Reply:
Yup that's the plan in a future PR for this refactor